content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -0,0 +1,169 @@
+for more information.
+
+    {{< text bash >}}
+    $ kubectl patch svc istio-ingressgateway -n istio-system -p '{"spec":{"externalTrafficPolicy":"Local"}}'
+    {{< /text >}}
+
+* Verify that the `httpbin` workload and ingress gateway are working as expected using this command.
+
+    {{< text bash >}}
+    $ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+* Verify the ingress gateway receives the original client source IP address, which
+will be used in the authorization policy in this task.
+
+    {{< text bash >}}
+    $ CLIENT_IP=$(curl $INGRESS_HOST/ip -s | grep "origin" | cut -d'"' -f 4) && echo $CLIENT_IP
+    105.133.10.12
+    {{< /text >}}
+
+{{< warning >}}
+If you don’t see the expected output as you follow the task, retry after a few seconds.
+Caching and propagation overhead can cause some delay.
+{{< /warning >}}
+
+## IP-based allow list and deny list
+
+1. The following command creates the authorization policy, `ingress-policy`, for
+the Istio ingress gateway. The following policy sets the `action` field to `ALLOW` to
+allow the IP addresses specified in the `ipBlocks` to access the ingress gateway,
+IP addresses not in the list will be denied, also known as whitelist-style access control.
+The `ipBlocks` supports both single IP address and CIDR notation.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
+    EOF
+    {{< /text >}}
+
+1. Verify that a request to the ingress gateway is denied.
+
+    {{< text bash >}}
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    403
+    {{< /text >}}
+
+1. Update the `ingress-policy` to include your client IP address.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: ingress-policy
+      namespace: istio-system
+    spec:
+      selector:
+        matchLabels:
+          app: istio-ingressgateway
+      action: ALLOW
+      rules:
+      - from:
+        - source:
+           ipBlocks: ["1.2.3.4", "5.6.7.0/24", "$CLIENT_IP"]
+    EOF
+    {{< /text >}}
+
+1. Verify that a request to the ingress gateway is allowed.
+
+    {{< text bash >}}
+    $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+    200
+    {{< /text >}}
+
+1. The following command updates the `ingress-policy` authorization policy to set
```

Comment:
Some peeps will find whitelist/blacklist offensive. If individuals really don't know what an allow or deny list is, the CRD documentation can be improved within the protobufs rather than spelled out in a slew of places all over the documentation. For these reasons, I'd recommend removing `also known as blacklist-style access control`. 

change:

`The following command updates the` -> `Update the ...`

`ingress gateway, ...` -> `ingress gateway:`.

Reply:
Here's more background on the whitelist/blacklist issue:
https://developers.google.com/style/word-list#blacklist

Reply:
Thanks for the suggestion and the reference, removed.